prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.html

```diff
@@ -89,10 +89,10 @@ <h6 class="mb-0">
       <ng-container appPageSubheaderTitle>
         {{
         (isInitialEnrolment)
-          ? 'Enrolment Review'
-          : (withinDaysOfRenewal)
-            ? 'Renew Enrolment'
-            : 'Enrolment Information'
+        ? 'Enrolment Review'
```

Comment:
Not sure why this indentation is removed but might want to check your config

Reply:
This is an example of where scanning/reviewing your own PR before making it available for review might catch bits of code like this that were staged and committed by accident.

Reply:
Not sure why this is happening for this piece only but when I hit save it always goes back.

Reply:
Fixed for now on github but my editor insists to put them on the same level